contracts/smart-contract-wallet/test/upgrades/v1/BaseSmartAccountV1.sol

```diff
@@ -33,7 +33,7 @@ struct FeeRefund {
  * This contract provides the basic logic for implementing the IAccount interface: validateUserOp function
  * Specific account implementation should inherit it and provide the account-specific logic
  */
-abstract contract BaseSmartAccount is IAccount, BaseSmartAccountErrors {
+abstract contract BaseSmartAccount is IAccount, BaseSmartAccountErrorsV1 {
```

Comment:
version number is confusing... current smart account file version says 2.0.0 
can we also not keep Errors.sol as it is?

Reply:
I had to create separate Errors file for the legacy V1 version of SA, as for the current SA version I removed several errors which are related to FF and are not used anymore in SmartAccount.sol

Version number [is 1.0.0 for SmartAccountV1.sol](https://github.com/bcnmy/scw-contracts/blob/7e4df55e10b82eb9fb9b9aba9774fde76b9d5706/contracts/smart-contract-wallet/test/upgrades/v1/SmartAccountV1.sol#L42)